### ISSUE TITLE
clarify input size value of softmax

### ISSUE THREAD

**Issue by atlaschiew** [Hiroaki-K4/multilayer_perceptron#14]
[    softmax_layer = Softmax(batch_size, 2)](https://github.com/Hiroaki-K4/multilayer_perceptron/blob/337abab0024d2000cf50ce237133f1726de697cd/srcs/train.py#L61)

hi sir, in the code above, what value `$input_size` suppose to be? 2 or `$batch_size`?

**Comment by Hiroaki-K4** [comment]
@atlaschiew 
It means the number of input data, so batch_size.
